contracts/StakeManager.sol

```diff
@@ -106,7 +106,7 @@ contract StakeManager is Ownable {
     /**
      * @notice Process epoch if it has ended
      */
-    modifier processEpoch() {
+    modifier finalizeEpoch() {
```

Comment:
Really wat this modifier does is finalizing the epoch if it's duration has passed.
There's an argument to be made that there's cases where it won't finalize, but I still think it's better than "process epoch" which can mean many things.